runtime/kusama/src/lib.rs

```diff
@@ -1531,31 +1530,11 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPalletsWithSystem,
-	(
-		SlotsCrowdloanIndexMigration,
-		pallet_staking::migrations::v9::InjectValidatorsIntoVoterList<Runtime>,
-	),
+	(pallet_staking::migrations::v9::InjectValidatorsIntoVoterList<Runtime>,),
```

Comment:
So this should be removed as well? (says [Kian here](https://github.com/paritytech/polkadot/issues/5354#issuecomment-1108184161))

Reply:
Indeed, i have a follow-up PR lined up. I can also pull that change in here if you'd like, though

Reply:
Yeah, let's keep them together.

Reply:
ok